### DISCUSSION TITLE
Apparmor profile

### DISCUSSION THREAD

**Discussion by ple1n** [oxen-io/lokinet#2149] [Q&A]
Do I need an apparmor profile for this, or, this question might sound retarded. In that case, how do I improve my security to a reasonable level, ie. least privilege, firejail ?

**Comment by majestrate** [comment]
it depends on your threat model.

a MAC framework like app armor can help limit the scope of a compromise if there is an exploitable bug in lokinet. such system layer security is an optional nice to have for most consumer grade threat models. when you need it you'd know a lot more on why you need it for your specific threat model.
